crates/storage/src/merkle/column.rs

```diff
@@ -56,7 +63,11 @@ where
             Self::MerkleDataColumn(column) => {
                 Self::MERKLE_DATA_COLUMNS_START.wrapping_add(column.as_u32())
             }
```

Comment:
I think we need to fix this one as well. Seems like the in-memory database implementation assumes these values are contiguous.

Reply:
nop, we don't need to fix these. the in memory db creates the columns based on the total count. anything in between is fine

Reply:
But `u16::MAX` isn't in-between though, or am I missing something here? `Self::MERKLE_DATA_COLUMNS_START` is a constant set to `u16::MAX` from what I see.